AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -110,10 +110,25 @@ public async Task<ActionResult<HearingDetailsResponse>> Post([FromBody] BookHear
 
                 if (request.IsMultiDay)
                 {
+
```

Comment:
To minimise how much this method grows, can you wrap the logic inside the isMultiDay block to its own method since the enclosing method doesn't rely on any of the variables produced